packages/world-plugins/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@latticexyz/world-plugins",
```

Comment:
can we mark this private for now? or otherwise "hide" this from the lerna release flow so canaries can work until we have an actual package here?